src/sentry/static/sentry/app/components/pluginConfig.jsx

```diff
@@ -81,6 +81,23 @@ const PluginConfig = React.createClass({
     });
   },
 
+  testPlugin() {
+    let loadingIndicator = IndicatorStore.add(t('Sending test..'));
+    this.api.request(this.getPluginEndpoint(), {
+      method: 'POST',
+      data: {
+        test: true
+      },
+      success: data => {
+        IndicatorStore.remove(loadingIndicator);
+        IndicatorStore.add(t('Test Complete!'), 'success');
+      },
+      error: error => {
+        IndicatorStore.add(t('Something is wrong. Please try again.'), 'error');
```

Comment:
i'd make it more clear that this is a problem with the request, not necessarily the test itself. maybe something like "An unexpected error occurred while testing your plugin. Please try again or contact support" ?